src/main/java/com/sreeram/bccleaderboard/models/Player.java

```diff
@@ -2,57 +2,26 @@
 
 import lombok.Data;
 
-import java.util.HashMap;
-import java.util.Map;
+import java.util.List;
 
 @Data
 public class Player {
+  
   private final String username;
 
-  // I feel combining the below tournament related details together would be good.
-  // For now, I am keeping them as separate fields.
-  /**
-   * key : value pairs
-   * tournament id : points scored in the tournament
-   */
-  private Map<String, Float> tmtPoints;
-
-  /**
-   * key : value pairs
-   * tournament id : tie-break score assigned in the tournament
-   */
-  private Map<String, Float> tiebreakScore;
-
-  /**
-   * key : value pairs
-   * tournament id : club score assigned in the tournament
-   */
-  private Map<String, Integer> clubScore;
-
-  public Player(String username) {
-    this.username = username;
-    tmtPoints = new HashMap<>();
-    tiebreakScore = new HashMap<>();
-    clubScore = new HashMap<>();
-  }
+  // outcomes of the player in each tournament he participated in the month
+  private List<PlayerTournamentOutcome> outcomes;
```

Comment:
I think outcomes is not needed as part of the response. Boils down to the same comment regarding player response.

Reply:
Hmm okay 👍